### ISSUE TITLE
Allow the setting of schemaUpdateOptions as provided by the Java Client API 

### ISSUE THREAD

**Issue by ASRagab** [GoogleCloudDataproc/spark-bigquery-connector#143]
There are two options that are provided for modifying the table schema through the Java Client API:
`SchemaUpdateOption.ALLOW_FIELD_RELAXATION` and `SchemaUpdateOption.ALLOW_FIELD_ADDITION`

It would be used in this way:
```java
      QueryJobConfiguration queryConfig =
          QueryJobConfiguration.newBuilder(query)
              .setUseLegacySql(false)       
              .setSchemaUpdateOptions(ImmutableList.of(SchemaUpdateOption.ALLOW_FIELD_RELAXATION))
              .setWriteDisposition(WriteDisposition.WRITE_APPEND)
              .setDestinationTable(tableId)
              .build();
```

Would it be possible to be able to set those schema update options through the spark-bigquery-connector, something like:

```scala
    df.write
      .format("bigquery")
      .option("table", table)
      .option("allowFieldAddition", "true") // proposed usage
     .option("allowFieldRelaxation", "true")
      .mode(SaveMode.Append) 
      .save()
```

Thank you for your consideration.

**Comment by ASRagab** [comment]
Thank you!

**Comment by davidrabinowitz** [comment]
Fixed in 0.14.0-beta
